context.go

```diff
@@ -162,3 +168,16 @@ func (c *context) ParamsInt(name string) int {
 	i, _ := strconv.Atoi(c.Params(name))
 	return i
 }
+
+func (c *context) Cookie(name string) string {
+	cookie, err := c.Request().Cookie(name)
+	if err != nil {
+		return ""
+	}
+
+	val, err := url.QueryUnescape(cookie.Value)
+	if err != nil {
+		return cookie.Value
+	}
```

Comment:
Why those errors being ignored instead of returning to the top?

Reply:
1. The first error is ignored because not able to get the cookie value simply means the cookie is not found in the request.
2. The second error is ignored because it is legitimate to have unescapable characters as part of the cookie value.